### DISCUSSION TITLE
Yolov8poseX : Training multiple classes on cropped boxes (100pixels) to be detected in 5k images

### DISCUSSION THREAD

**Discussion by AdwaitMD** [ultralytics/ultralytics#15613] [Q&A]
Hi, 
I have training data for 12 classes (with key points) [approx 5000 images per class. The training images are, on average, 100*100 pixels. However, if the yolov8poseX model is trained on this data, will it be able to detect different classes of 100*100 pixels present in a large 5K image? Or does training data need to be of 5K images with multiple small classes present in it? Any tips for this task would be useful.

**Comment by glenn-jocher** [comment]
@AdwaitMD training on 100x100 pixel images should work, but ensure your model's input size is appropriate for detecting small objects in larger images. You might need to adjust the image size during training to improve detection accuracy. For more detailed tips, visit the Ultralytics documentation on best training practices.
